### PR TITLE
Adding desired state

### PR DESCRIPTION
**Pull request by mbfrahry**

### CODE REVIEW

**Comment by mbfrahry** [comment]
Tests passing 

image

**Comment by mbfrahry** [comment]
This is ready for another review @scross01

File: website/docs/r/oraclepaas_database_service_instance.html.markdown [hunk 1]
```diff
@@ -61,7 +61,7 @@ The following arguments are supported:
 * `default_access_rules` - (Optional) Specifies the details on which default access rules are enable or disabled. Default Access Rules
 are configured below.
 
-* `desired_state` - (Optional) Specifies the desired state of the servie instance. Allowed values are `start`, `stop`,
+* `desired_state` - (Optional) Specifies the desired state of the service instance. Allowed values are `start`, `stop`,
```
Comment (scross01) [inline website/docs/r/oraclepaas_database_service_instance.html.markdown]: should change to `running` and `shutdown`

However, I just noticed we added the DB running state in v1.1.0, so this is already out there. Could we "deprecate" the use of start/stop (with a warning) and add running/shutdown as the preferred values.

File: website/docs/r/oraclepaas_java_service_instance.html.markdown [hunk 2]
```diff
@@ -128,6 +128,8 @@ value is `PAAS`.
 * `backup_destination` - (Optional) Specifies whether to enable backups for this Oracle Java Cloud Service instance.
 Valid values are `BOTH` or `NONE`. Defaults to `BOTH`.
 
+* `desired_state` - (Optional) Specifies the desired state of the service instance. Allowed values are `start` or `stop`.
```
Comment (scross01) [inline website/docs/r/oraclepaas_java_service_instance.html.markdown]: should be `running` and `shutdown`

**Pull request merged**